content/overview/quick-start.md

```diff
@@ -19,13 +19,13 @@ You can find the source code for Haystack on GitHub. This is also the main chann
 
 ## Installation
 
-These are the instructions for installing Haystack. The most straightforward way to install the latest release of Haystack is through [pip](https://github.com/pypa/pip).
+Use [pip](https://github.com/pypa/pip) to install the latest Haystack release:
 
 {{< tabs totalTabs="3">}}
 
 {{< tab tabName="Minimal"  >}}
 
-This command installs everything needed for basic Pipelines that use an InMemoryDocumentStore and external LLM provider (e.g. OpenAI).
+This command installs everything needed for basic Pipelines that use an InMemoryDocumentStore and an external LLM provider (for example, OpenAI).
```

Comment:
In the descriptions of each method, I'd focus on which one to choose. If you're thinking of someone who's new, they may not know what "basic pipelines" or "inmemorydocumentstore" are. So maybe try focusing on the tasks they can do with each installation. 

For example, for minimal, you could say: Use this installation method to try Haystack out. It contains the basic features blahblahblah